modules/ssl/ssl_engine_pphrase.c

```diff
@@ -118,6 +118,16 @@ static apr_file_t *writetty = NULL;
 static apr_file_t *readtty = NULL;
 
 int ssl_pphrase_Handle_CB(char *, int, int, void *);
+#if defined(HAVE_OPENSSL_ENGINE_H) && defined(HAVE_ENGINE_INIT)
+int passphrase_ui_open(UI *ui);
```

Comment:
These don't seem to be defined in a header, or used outside this file. Shouldn't they be static?

Reply:
Yes, all the UI related functions should be static.